packages/fx-core/src/plugins/resource/utils4v2.ts

```diff
@@ -147,20 +142,80 @@ export async function deployAdapter(
   return ok({ output: deployOutput });
 }
 
-export async function provisionLocalResourceAdapter( ctx: Context, tokenProvider: TokenProvider, plugin: Plugin & ArmResourcePlugin) : Promise<Result<LocalSetting, FxError>>{
+export async function provisionLocalResourceAdapter(
+  ctx: Context,
+  inputs: Inputs,
+  localSettings: LocalSettings,
+  tokenProvider: TokenProvider,
+  plugin: Plugin & ArmResourcePlugin
+): Promise<Result<LocalSettings, FxError>> {
   if (!plugin.localDebug) return err(PluginHasNoTaskImpl(plugin.displayName, "localDebug"));
-  //TODO
-  throw new Error();
+  const pluginContext: PluginContext = convert2PluginContext(ctx, inputs);
+  pluginContext.localSettings = {
+    teamsApp: ConfigMap.fromJSON(localSettings.teamsApp)!,
+    auth: ConfigMap.fromJSON(localSettings.auth),
+    backend: ConfigMap.fromJSON(localSettings.backend),
+    bot: ConfigMap.fromJSON(localSettings.bot),
+    frontend: ConfigMap.fromJSON(localSettings.frontend),
+  };
+  pluginContext.appStudioToken = tokenProvider.appStudioToken;
+  pluginContext.azureAccountProvider = tokenProvider.azureAccountProvider;
+  pluginContext.graphTokenProvider = tokenProvider.graphTokenProvider;
+  const res = await plugin.localDebug(pluginContext);
```

Comment:
where is localDebug defined?

Reply:
provisionLocalResources is replacement of localDebug

Reply:
I mean plugin.localDebug here. I can not find the implementation.